roles/ood_user_reg/tasks/main.yml

```diff
@@ -66,3 +66,29 @@
 - name: Build the updated Apache config
   command: /opt/ood/ood-portal-generator/sbin/update_ood_portal
   ignore_errors: yes
+
+- name: Create RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo id -u {{ RegUser_app_user }} &>/dev/null || sudo useradd -c "{{ RegUser_app_user_full_name }}" {{ RegUser_app_user }}'
+  become_user: vagrant
+
+- name : delay
+  command: sleep 2
+
+- name: Change passwd RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo echo "{{ RegUser_app_user_passwd }}" | sudo passwd --stdin "{{ RegUser_app_user }}"'
+  become_user: vagrant
+
+- name: Create Cluster keys for RegUser - {{ RegUser_app_user }} on OHPC
+  shell: ssh ohpc 'sudo -s su - reggie -c "sh /usr/bin/cluster-env"'
```

Comment:
Should be able to trim this to just `su - reggie -c "/usr/bin/cluster-env"`

Reply:
i thought we are just logging and exit out? e.g. `su - reggie exit` ? Do we have to execute the script explicitly?

Reply:
yeah but i think that did not work.